tests/infra/rates.py

```diff
@@ -7,7 +7,7 @@
 
 from loguru import logger as LOG
 
-COMMIT_COUNT_CUTOFF = 10
+COMMIT_COUNT_CUTOFF = 4
```

Comment:
I had to bump this up after my latest changes on https://github.com/microsoft/CCF/pulls?q=is%3Apr+is%3Aclosed as the signature forwarding test took some time to generate the signed requests. Is it faster now that we use the right curve?

Reply:
I've used your 15 for now. I'd really like to remove this `process_next` loop entirely, but haven't found a workable alternative yet. I think the `remote` should be able to indicate when it thinks it has finished, perhaps by grepping stdout again, and we should only need to wait briefly for commit stability after that.